dubbo-metadata/dubbo-metadata-report-redis/src/main/java/org/apache/dubbo/metadata/store/redis/RedisMetadataReport.java

```diff
@@ -207,4 +207,8 @@ private String getMetadataStandalone(BaseMetadataIdentifier metadataIdentifier)
         }
     }
 
+    @Override
+    public boolean registerServiceAppMapping(String serviceInterface, String defaultMappingGroup, String newConfigContent, Object ticket) {
+        throw new UnsupportedOperationException("Redis metadataReport implementation does not support registerServiceAppMapping() method.");
```

Comment:
To fix https://github.com/apache/dubbo/issues/12627, you should use jedis to set content here.

Reply:
PTAL